src/style.css

```
@@ -0,0 +1,121 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir-Book, sans-serif;
+  color: rgba(61, 78, 97, 1);
+  font-weight: 300;
+  font-size: 16px;
+}
+
+/* body {
+  margin: 20px 10px 0;
+} */
+
+.form {
+  /* display: inline-block; */
+  width: 100%;
+  margin-top: 20px;
+  background: #fff;
+}
+
+.bigInput {
+  display: inline-block;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  /* padding: 24px 0;
+  line-height: 20px; */
+}
```

structure your styles according to what they do, it will be easier to work with

```suggestion
.bigInput {
  display: inline-block;
  width: 100%;
  height: 70px;
  padding-left: 62px;
  
  border: 1px solid #e1e7ed;
  border-radius: 4px;
  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
}
```